From 91f3fa8b7473438dc6c8b7e314ccf14336ec1dd9 Mon Sep 17 00:00:00 2001
From: Hao Shi <hao.shi@amlogic.com>
Date: Mon, 12 Jun 2023 09:40:46 +0000
Subject: [PATCH] zapper: close v4l2-m2m in zapper [1/2]

PD#SWPL-118958

Problem:
Zapper 2K S1A - 128M DDR - optimization v4l2.

Solution:
close V4L2_MEM2MEM_DEV & CONFIG_AMLOGIC_VIDEOBUF_RESOURCE.

Verify:
aq2432

Change-Id: I7ea4b2c5baababa6cf5743c6c5dabad2ab6ee925
Signed-off-by: Hao Shi <hao.shi@amlogic.com>
---
 init/Kconfig.gki | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/Kconfig.gki b/init/Kconfig.gki
index ce622014516e..ad790c0803ce 100644
--- a/init/Kconfig.gki
+++ b/init/Kconfig.gki
@@ -89,7 +89,7 @@ config GKI_HIDDEN_QCOM_CONFIGS
 config GKI_HIDDEN_MEDIA_CONFIGS
 	bool "Hidden Media configs needed for GKI"
 	select VIDEOBUF2_CORE
-	select V4L2_MEM2MEM_DEV
+	select V4L2_MEM2MEM_DEV if ! AMLOGIC_ZAPPER_CUT
 	select MEDIA_CONTROLLER
 	select MEDIA_CONTROLLER_REQUEST_API
 	select MEDIA_SUPPORT
-- 
2.25.1

